tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -549,25 +558,39 @@ describe(__filename, () => {
       ];
       dispatchAddon();
       dispatchAddonReviews({ reviews });
+
       const root = render();
-      expect(root.find(Paginate)).toHaveProp('count', reviews.length);
+      const footer = root.find('.AddonReviewList-reviews').prop('footer');
+      const paginator = shallow(footer);
+
+      expect(paginator).toHaveProp('count', reviews.length);
     });
 
     it('sets the paginator to page 1 without a query', () => {
       dispatchAddon();
       dispatchAddonReviews();
+
       // Render with an empty query string.
       const root = render({ location: fakeRouterLocation() });
-      expect(root.find(Paginate)).toHaveProp('currentPage', 1);
+      const footer = root.find('.AddonReviewList-reviews').prop('footer');
+      const paginator = shallow(footer);
+
+      expect(paginator).toHaveProp('currentPage', 1);
     });
 
     it('sets the paginator to the query string page', () => {
+      const page = 3;
+
       dispatchAddon();
       dispatchAddonReviews();
+
       const root = render({
-        location: fakeRouterLocation({ query: { page: 3 } }),
+        location: fakeRouterLocation({ query: { page } }),
       });
-      expect(root.find(Paginate)).toHaveProp('currentPage', 3);
+      const footer = root.find('.AddonReviewList-reviews').prop('footer');
+      const paginator = shallow(footer);
```

Comment:
Is it worth creating a paginator variable when you're only using it once below?

Reply:
Ok, got it. I'll remove the variable and add a comment instead 😅